python/ray/experimental/serve/queues.py

```diff
@@ -0,0 +1,155 @@
+from collections import defaultdict, deque
+from typing import Any
+
+import numpy as np
+from dataclasses import dataclass
+
+import ray
+from ray.experimental.serve.utils import get_custom_object_id, logger
+
+
+@dataclass
+class Query:
+    request_body: Any
+    result_oid: ray.ObjectID
+
+    @staticmethod
+    def new(req: Any):
+        return Query(request_body=req, result_oid=get_custom_object_id())
+
+
+@dataclass
+class WorkIntent:
+    work_oid: ray.ObjectID
+
+    @staticmethod
+    def new():
+        return WorkIntent(work_oid=get_custom_object_id())
+
+
+class CentralizedQueues:
+    """A "router" that routes request to available workers.
```

Comment:
Why is router in quotes? If it isn't a router by standard definition, don't use the word.